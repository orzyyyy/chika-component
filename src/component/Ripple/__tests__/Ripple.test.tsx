import React from 'react';
import { mount } from 'enzyme';
import 'nino-cli/scripts/setup';
let Ripple;
switch (process.env.LIB_DIR) {
  case 'lib':
    Ripple = require('../../../../lib/component/Ripple').default;
    break;
  case 'dist':
    Ripple = require('../../../../dist/lib/Ripple').default;
    break;
  default:
    Ripple = require('..').default;
    break;
}

describe('Ripple', () => {
  it('render correctly', () => {
    const wrapper = mount(<Ripple />);
    expect(wrapper).toMatchSnapshot();
  });
});
