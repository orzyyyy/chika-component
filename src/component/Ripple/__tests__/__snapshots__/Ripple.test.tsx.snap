// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ripple render correctly 1`] = `
<Ripple>
  <div>
    <div
      className="Ripple"
      style={Object {}}
    />
  </div>
</Ripple>
`;
